src/Chirp.Core/CheepRepository.cs

```diff
@@ -17,7 +19,13 @@ public CheepRepository(ChirpDBContext context)
     {
         _dbcontext = context;
     }
-
+    /// <summary>
+    /// Method <c>GetCheeps</c> returns a list of Cheeps.
```

Comment:
I may be pedantic, but I feel that it should be stated where the cheeps come from (the database). The return type is not really relevant in a comment, since it is explicit in the signature.